test/queue/queueservice_test.js

```diff
@@ -164,4 +171,70 @@ suite('queue/QueueService', function() {
     debug("pending message count: %j", count);
     assert(typeof(count) === 'number', "Expected count as number!");
   });
+
+  test("deleteUnusedWorkerQueues (can delete queues)", async () => {
+    // 11 days into the future, so we'll delete all queues (yay)
+    let now = new Date(Date.now() + 11 * 24 * 60 * 60 * 1000);
+    let deleted = await queueService.deleteUnusedWorkerQueues(now);
+    // There should always be at least one queue, because test cases above
```

Comment:
So.. this is a nit at least for me, but relying on data to be setup by previous tests makes it difficult to run this specific test in isolation and depends on the sequence of other tests.